packages/react-native/src/primitives/ErrorMessage/ErrorMessage.tsx

```diff
@@ -0,0 +1,51 @@
+import React, { useState } from 'react';
+import { Text, View } from 'react-native';
+
+import { icons } from '../../assets';
+import { Icon } from '../Icon';
+import { IconButton } from '../IconButton';
+
+import { ErrorMessageProps } from './types';
+import { styles } from './styles';
+
+export default function Label({
+  accessibilityRole = 'alert',
+  children,
```

Comment:
Should this component have `children`?

Reply:
Instead of `children`, should `ErrorMessage` just accept an `errorMessage` prop of type `string`?